src/main/java/Duke.java

```diff
@@ -1,10 +1,225 @@
+import java.io.*;
```

Comment:
Imported classes should always be listed explicitly, avoid using `import java.io.*`